src/components/flashcard/flashcard.tsx

```diff
@@ -105,7 +105,7 @@ export const Flashcard = ({
   }
 
   function isCardTopInViewPort(cardDiv: HTMLDivElement) {
-    const { top } = cardDiv.getBoundingClientRect();
-    return top <= (window.innerHeight || document.documentElement.clientHeight);
+    const { top, bottom } = cardDiv.getBoundingClientRect();
+    return bottom >= 0 || top <= (window.innerHeight || document.documentElement.clientHeight);
```

Comment:
scroll wouldn't work anymore, this should be `&&` (since `bottom >= 0` is always true)

and while you're at it, can we rename this method to `isCardInViewport` to be more concise?

Reply:
haha, it's been a long day, thanks for catching that😂